### PR TITLE
Fix integration tests and integration cli to run on older versions

### PR DESCRIPTION
**Pull request by AntaresS**
**- What I did**
Fix `integration` tests and `integration-cli` tests to run on older versions (e.g. 17.06-ee) with @tiborvass 

**- How I did it**
Add version requirements in some failed tests and skip tests that are not runnable in older versions.

**- How to verify it**
Run the whole test suit accross a version/platform matix. 

**- Description for the changelog**

Fixing failed `integration` and `integration-cli` tests against older engine versions. 

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by AntaresS** [comment]
rebase in progress...

File: pkg/term/proxy_test.go [hunk 1]
```diff
@@ -10,15 +10,41 @@ import (
 )
 
 func TestEscapeProxyRead(t *testing.T) {
-	escapeKeys, _ := ToBytes("DEL")
-	keys, _ := ToBytes("a,b,c,+")
+	escapeKeys, _ := ToBytes("")
```
Comment (tiborvass) [inline pkg/term/proxy_test.go]: @AntaresS Why these changes?

File: integration-cli/docker_api_network_test.go [hunk 2]
```diff
@@ -60,14 +61,27 @@ func (s *DockerSuite) TestAPINetworkCreateCheckDuplicate(c *check.C) {
 	}
 
 	// Creating a new network first
-	createNetwork(c, configOnCheck, http.StatusCreated)
+	expectedStatus := http.StatusCreated
+	isOlderAPI := versions.LessThan(testEnv.DaemonAPIVersion(), "1.34")
+	if isOlderAPI {
+		expectedStatus = http.StatusCreated
```
Comment (thaJeztah) [inline integration-cli/docker_api_network_test.go]: Both old and new API have the same status code here?

File: integration-cli/docker_api_containers_test.go [hunk 3]
```diff
@@ -724,13 +725,21 @@ func (s *DockerSuite) TestContainerAPIVerifyHeader(c *check.C) {
 	// Try with no content-type
 	res, body, err := create("")
 	c.Assert(err, checker.IsNil)
-	c.Assert(res.StatusCode, checker.Equals, http.StatusBadRequest)
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		c.Assert(res.StatusCode, checker.Equals, http.StatusInternalServerError)
+	}else {
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: formatting of `else` - needs a space

File: integration-cli/docker_api_containers_test.go [hunk 4]
```diff
@@ -724,13 +725,21 @@ func (s *DockerSuite) TestContainerAPIVerifyHeader(c *check.C) {
...
+	}
 	body.Close()
 
 	// Try with wrong content-type
 	res, body, err = create("application/xml")
 	c.Assert(err, checker.IsNil)
-	c.Assert(res.StatusCode, checker.Equals, http.StatusBadRequest)
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		c.Assert(res.StatusCode, checker.Equals, http.StatusInternalServerError)
+	}else {
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: same here (space)

File: integration-cli/docker_api_containers_test.go [hunk 5]
```diff
@@ -724,13 +725,21 @@ func (s *DockerSuite) TestContainerAPIVerifyHeader(c *check.C) {
 	// Try with no content-type
 	res, body, err := create("")
 	c.Assert(err, checker.IsNil)
-	c.Assert(res.StatusCode, checker.Equals, http.StatusBadRequest)
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		c.Assert(res.StatusCode, checker.Equals, http.StatusInternalServerError)
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: Wonder if we should actually backport the fix for this status; 500 may be due to a bug / not properly handled condition

File: integration-cli/docker_api_containers_test.go [hunk 6]
```diff
@@ -756,7 +765,11 @@ func (s *DockerSuite) TestContainerAPIInvalidPortSyntax(c *check.C) {
 
 	res, body, err := request.Post("/containers/create", request.RawString(config), request.JSON)
 	c.Assert(err, checker.IsNil)
-	c.Assert(res.StatusCode, checker.Equals, http.StatusBadRequest)
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		c.Assert(res.StatusCode, checker.Equals, http.StatusInternalServerError)
+	}else {
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: same here (space)

File: integration-cli/docker_api_containers_test.go [hunk 7]
```diff
@@ -776,7 +789,11 @@ func (s *DockerSuite) TestContainerAPIRestartPolicyInvalidPolicyName(c *check.C)
...
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: same here

File: integration-cli/docker_api_containers_test.go [hunk 8]
```diff
@@ -796,7 +813,11 @@ func (s *DockerSuite) TestContainerAPIRestartPolicyRetryMismatch(c *check.C) {
...
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: same here

File: integration-cli/docker_api_containers_test.go [hunk 9]
```diff
@@ -816,7 +837,11 @@ func (s *DockerSuite) TestContainerAPIRestartPolicyNegativeRetryCount(c *check.C
...
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: same here (couple more below)

File: integration-cli/docker_api_containers_test.go [hunk 10]
```diff
@@ -1910,8 +1950,13 @@ func (s *DockerSuite) TestContainersAPICreateMountsCreate(c *check.C) {
 	}
 
 	var selinuxSharedLabel string
-	if runtime.GOOS == "linux" {
-		selinuxSharedLabel = "z"
+	// this test label was added after a bug fix in 1.32, thus add requirements min API >= 1.32
```
Comment (thaJeztah) [inline integration-cli/docker_api_containers_test.go]: may want to look at backporting the fix

**Review by thaJeztah — CHANGES_REQUESTED** [review]
didn't do a full review; just some things that stood out

File: integration-cli/docker_api_containers_test.go (hunk 5 above)
Comment (tiborvass) [reply integration-cli/docker_api_containers_test.go]: @AntaresS agree that we should look at what the old test was testing, I doubt we were testing for 500.

File: integration-cli/docker_api_containers_test.go (hunk 10 above)
Comment (tiborvass) [reply integration-cli/docker_api_containers_test.go]: @thaJeztah not in this PR

File: integration-cli/docker_api_containers_test.go (hunk 5 above)
Comment (thaJeztah) [reply integration-cli/docker_api_containers_test.go]: Hm, right, it's a bit tricky, because it's effectively not related to the API version, but to the version of the daemon that's running (i.e., the "internal server error" was caused by an unhandled error condition, so defaulted to a 500). i.e. using API version 1.31 (or older) on a _current_ version of the daemon would still have the fix, so will return `http.StatusBadRequest`.

Was thinking if we should do a `statuscode != 200` or `status code >= 400`, but then we'd miss possible regressions if a `500` is returned on the current codebase.

File: integration-cli/docker_api_build_test.go [hunk 11]
```diff
@@ -406,6 +407,10 @@ func (s *DockerSuite) TestBuildAddRemoteNoDecompress(c *check.C) {
 }
 
 func (s *DockerSuite) TestBuildChownOnCopy(c *check.C) {
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.31") {
```
Comment (vdemeester) [inline integration-cli/docker_api_build_test.go]: could be a onliner : 
```go
skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.31"), "new feature added in 1.31, go version `s", testEnv.DaemonAPIVersion())
```

(not even sure we need to print the current version)

File: integration-cli/docker_api_containers_test.go (hunk 5 above)
Comment (vdemeester) [reply integration-cli/docker_api_containers_test.go]: I tend to agree with @thaJeztah… if it's not related to the API version we shouldn't check the API version.. We could also check the `dockerd` version number (or the version field in the `info` endpoint for a cleaner way of doing the check) and act based on that ?

File: integration-cli/docker_api_logs_test.go [hunk 12]
```diff
@@ -149,6 +150,9 @@ func (s *DockerSuite) TestLogsAPIUntilFutureFollow(c *check.C) {
 }
 
 func (s *DockerSuite) TestLogsAPIUntil(c *check.C) {
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.34") {
```
Comment (vdemeester) [inline integration-cli/docker_api_logs_test.go]: Same as above for oneliner :angel:

File: integration-cli/docker_api_network_test.go [hunk 13]
```diff
@@ -64,7 +65,12 @@ func (s *DockerSuite) TestAPINetworkCreateCheckDuplicate(c *check.C) {
 	c.Assert(isNetworkAvailable(c, name), checker.Equals, true)
 
 	// Creating another network with same name and CheckDuplicate must fail
-	createNetwork(c, configOnCheck, http.StatusConflict)
+	isOlderAPI := versions.LessThan(testEnv.DaemonAPIVersion(), "1.34")
+	expectedStatus := http.StatusConflict
```
Comment (vdemeester) [inline integration-cli/docker_api_network_test.go]: Is it API based or docker daemon version dependent ?

File: integration-cli/docker_cli_ps_test.go [hunk 14]
```diff
@@ -200,10 +201,19 @@ func (s *DockerSuite) TestPsListContainersFilterStatus(c *check.C) {
...
 	result := cli.Docker(cli.Args("ps", "-a", "-q", "--filter=status=rubbish"), cli.WithTimeout(time.Second*60))
-	result.Assert(c, icmd.Expected{
-		ExitCode: 1,
-		Err:      "Invalid filter 'status=rubbish'",
-	})
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+
+		result.Assert(c, icmd.Expected{
+			ExitCode: 1,
+			Err:      "Unrecognised filter value for status: rubbish",
```
Comment (vdemeester) [inline integration-cli/docker_cli_ps_test.go]: We could have only one assertion and just check and have the `Err` in a variable :angel:

File: integration-cli/docker_cli_ps_test.go [hunk 15]
```diff
@@ -849,7 +859,11 @@ func (s *DockerSuite) TestPsListContainersFilterPorts(c *check.C) {
 
 func (s *DockerSuite) TestPsNotShowLinknamesOfDeletedContainer(c *check.C) {
 	testRequires(c, DaemonIsLinux)
-
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.31") {
```
Comment (vdemeester) [inline integration-cli/docker_cli_ps_test.go]: Same as above for oneliner :angel:

File: integration-cli/docker_cli_start_test.go [hunk 16]
```diff
@@ -103,7 +104,11 @@ func (s *DockerSuite) TestStartPausedContainer(c *check.C) {
 	// an error should have been shown that you cannot start paused container
 	c.Assert(err, checker.NotNil, check.Commentf("out: %s", out))
 	// an error should have been shown that you cannot start paused container
-	c.Assert(out, checker.Contains, "cannot start a paused container, try unpause instead")
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		c.Assert(out, checker.Contains, "Cannot start a paused container, try unpause instead")
+	} else {
+		c.Assert(out, checker.Contains, "cannot start a paused container, try unpause instead")
```
Comment (vdemeester) [inline integration-cli/docker_cli_start_test.go]: :sweat:

**Review by vdemeester — CHANGES_REQUESTED** [review]

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: integration-cli/docker_api_build_test.go (hunk 11 above)
Comment (vdemeester) [reply integration-cli/docker_api_build_test.go]: oh right but it's with `check` so might not work 😭

File: integration-cli/requirements_test.go [hunk 17]
```diff
@@ -109,6 +117,9 @@ func Network() bool {
 }
 
 func Apparmor() bool {
+	if strings.HasPrefix(testEnv.DaemonInfo.OperatingSystem, "SUSE Linux Enterprise Server ") {
```
[the remark this answers is not in the document]
Comment (tiborvass) [reply integration-cli/requirements_test.go]: @thaJeztah yes. I asked @andrewhsu and it seems that we don't really support apparmor stuff on SUSE.

[the remark this answers is not in the document]
Comment (thaJeztah) [reply integration-cli/requirements_test.go]: In that case; should we disable it on the host?

File: integration-cli/docker_cli_start_test.go (hunk 16 above)
Comment (AntaresS) [reply integration-cli/docker_cli_start_test.go]: @vdemeester lolz..

File: integration-cli/docker_api_build_test.go (hunk 11 above)
Comment (AntaresS) [reply integration-cli/docker_api_build_test.go]: @vdemeester bingo. skip.If won't work here. They use two different interfaces.

File: integration-cli/docker_api_logs_test.go (hunk 12 above)
Comment (AntaresS) [reply integration-cli/docker_api_logs_test.go]: Same as the reply above 😸

File: integration-cli/docker_api_build_test.go (hunk 11 above)
Comment (vdemeester) [reply integration-cli/docker_api_build_test.go]: @AntaresS yep sorry about that, was thinking it was on `integration` tests :stuck_out_tongue:

File: integration-cli/docker_cli_ps_test.go (hunk 14 above)
Comment (AntaresS) [reply integration-cli/docker_cli_ps_test.go]: 👍

File: integration-cli/docker_cli_netmode_test.go [hunk 18]
```diff
@@ -46,7 +47,11 @@ func (s *DockerSuite) TestNetHostname(c *check.C) {
 	c.Assert(out, checker.Contains, runconfig.ErrConflictNetworkHostname.Error())
 
 	out, _ = dockerCmdWithFail(c, "run", "--net=container", "busybox", "ps")
-	c.Assert(out, checker.Contains, "Invalid network mode: invalid container format container:<name|id>")
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.31") {
+		c.Assert(out, checker.Contains, "--net: invalid net mode: invalid container format container:<name|id>")
```
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_cli_netmode_test.go]: @thaJeztah That's actually true. Now I wonder why was it even changed from `invalid` to `Invalid`. 
Aren't we following the error string convention -https://github.com/golang/go/wiki/CodeReviewComments#error-strings.
But the changes were made in [#33320](https://github.com/moby/moby/commit/92291a7355dff35be70b5d0ebf634f0232ca6a94#diff-83a075daa1f1fdbcfc9e8ff2364b0788R48).

File: integration-cli/docker_cli_ps_test.go [hunk 19]
```diff
@@ -200,10 +201,19 @@ func (s *DockerSuite) TestPsListContainersFilterStatus(c *check.C) {
 	c.Assert(RemoveOutputForExistingElements(containerOut, existingContainers), checker.Equals, secondID)
 
 	result := cli.Docker(cli.Args("ps", "-a", "-q", "--filter=status=rubbish"), cli.WithTimeout(time.Second*60))
-	result.Assert(c, icmd.Expected{
-		ExitCode: 1,
-		Err:      "Invalid filter 'status=rubbish'",
-	})
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+
```
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_cli_ps_test.go]: 👍

File: integration-cli/docker_cli_ps_test.go (hunk 15 above)
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_cli_ps_test.go]: 👍

File: integration-cli/docker_cli_netmode_test.go (hunk 18 above)
[the remark this answers is not in the document]
Comment (thaJeztah) [reply integration-cli/docker_cli_netmode_test.go]: Good question; we should probably change that back

File: integration-cli/docker_cli_start_test.go (hunk 16 above)
[the remark this answers is not in the document]
Comment (thaJeztah) [reply integration-cli/docker_cli_start_test.go]: SGTM

File: integration-cli/docker_cli_links_test.go [hunk 20]
```diff
@@ -28,7 +29,11 @@ func (s *DockerSuite) TestLinksInvalidContainerTarget(c *check.C) {
 	// an invalid container target should produce an error
 	c.Assert(err, checker.NotNil, check.Commentf("out: %s", out))
 	// an invalid container target should produce an error
-	c.Assert(out, checker.Contains, "could not get container")
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		c.Assert(out, checker.Contains, "Could not get container")
+	} else {
+		c.Assert(out, checker.Contains, "could not get container")
```
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_cli_links_test.go]: Yes. Lowercase `out` sounds better.

File: integration-cli/docker_cli_inspect_test.go [hunk 21]
```diff
@@ -463,5 +464,9 @@ func (s *DockerSuite) TestInspectInvalidReference(c *check.C) {
 	// This test should work on both Windows and Linux
 	out, _, err := dockerCmdWithError("inspect", "FooBar")
 	c.Assert(err, checker.NotNil)
-	c.Assert(out, checker.Contains, "no such image: FooBar")
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		c.Assert(out, checker.Contains, "Error: No such object: FooBar")
+	} else {
+		c.Assert(out, checker.Contains, "no such image: FooBar")
```
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_cli_inspect_test.go]: 👍

File: integration-cli/docker_cli_commit_test.go [hunk 22]
```diff
@@ -121,12 +122,17 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
...
-	// The ordering here is due to `PATH` being overridden from the container's
-	// ENV.  On windows, the container doesn't have a `PATH` ENV variable so
-	// the ordering is the same as the cli.
-	expectedEnv := "[PATH=/foo DEBUG=true test=1]"
-	if testEnv.OSType == "windows" {
-		expectedEnv = "[DEBUG=true test=1 PATH=/foo]"
+	expectedEnv := "[DEBUG=true test=1 PATH=/foo]"
+	// bug fixed in 1.36, add min APi >= 1.36 requirement
+	// PR record https://github.com/moby/moby/pull/35582
+	if !versions.LessThan(testEnv.DaemonAPIVersion(), "1.36") {
```
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_cli_commit_test.go]: 👍

File: integration-cli/docker_deprecated_api_v124_test.go [hunk 23]
```diff
@@ -24,12 +25,16 @@ func (s *DockerSuite) TestDeprecatedContainerAPIStartHostConfig(c *check.C) {
 	}
 	res, body, err := request.Post("/containers/"+name+"/start", request.JSONBody(config))
 	c.Assert(err, checker.IsNil)
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
```
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_deprecated_api_v124_test.go]: 👍

File: integration-cli/requirements_test.go (hunk 17 above)
[the remark this answers is not in the document]
Comment (tiborvass) [reply integration-cli/requirements_test.go]: @thaJeztah I agree it's not ideal. Maybe we could check the existence of the apparmor profile for docker instead of just checking whether apparmor is enabled. Do you think we can do that in a followup?

File: integration-cli/docker_api_containers_test.go (hunk 5 above)
Comment (tiborvass) [reply integration-cli/docker_api_containers_test.go]: @vdemeester @thaJeztah how about we add a TODO explaining the problem. I agree with both of you, there's just not a great way of doing order comparisons between dockerd versions. We can find a proper fix in a followup?

File: integration-cli/docker_api_network_test.go [hunk 24]
```diff
@@ -64,7 +65,12 @@ func (s *DockerSuite) TestAPINetworkCreateCheckDuplicate(c *check.C) {
...
+	if isOlderAPI {
+		expectedStatus = -http.StatusCreated
```
[the remark this answers is not in the document]
Comment (tiborvass) [reply integration-cli/docker_api_network_test.go]: @AntaresS would you mind adding an explanatory comment for the negative value, both here...

File: integration-cli/docker_api_network_test.go [hunk 25]
```diff
@@ -320,9 +334,13 @@ func createNetwork(c *check.C, config types.NetworkCreateRequest, expectedStatus
 	c.Assert(err, checker.IsNil)
 	defer resp.Body.Close()
 
-	c.Assert(resp.StatusCode, checker.Equals, expectedStatusCode)
+	if expectedStatusCode >= 0 {
+		c.Assert(resp.StatusCode, checker.Equals, expectedStatusCode)
+	} else {
+		c.Assert(resp.StatusCode, checker.Not(checker.Equals), -expectedStatusCode)
```
[the remark this answers is not in the document]
Comment (tiborvass) [reply integration-cli/docker_api_network_test.go]: @AntaresS ... and here.

File: integration-cli/docker_api_network_test.go (hunk 24 above)
[the remark this answers is not in the document]
Comment (AntaresS) [reply integration-cli/docker_api_network_test.go]: @tiborvass yes/

File: integration-cli/docker_api_containers_test.go (hunk 5 above)
Comment (tiborvass) [reply integration-cli/docker_api_containers_test.go]: @AntaresS ok let's fix this by only asserting the correct error, if API ≥ 1.32.

File: integration-cli/docker_cli_start_test.go [hunk 26]
```diff
@@ -103,7 +104,11 @@ func (s *DockerSuite) TestStartPausedContainer(c *check.C) {
 	// an error should have been shown that you cannot start paused container
 	c.Assert(err, checker.NotNil, check.Commentf("out: %s", out))
 	// an error should have been shown that you cannot start paused container
-	c.Assert(out, checker.Contains, "cannot start a paused container, try unpause instead")
+	expected := "cannot start a paused container, try unpause instead"
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.32") {
+		expected = "Cannot start a paused container, try unpause instead"
+	}
+	c.Assert(out, checker.Contains, expected)
```
Comment (thaJeztah) [inline integration-cli/docker_cli_start_test.go]: use `strings.ToLower()` here

File: integration-cli/docker_api_logs_test.go (hunk 12 above)
Comment (thaJeztah) [reply integration-cli/docker_api_logs_test.go]: Could be;

```
testRequires(c, MinimumAPIVersion("1.34"))
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Two minor nits, and it's probably good to add TODO's (as discussed above), but LGTM after those are done

File: integration-cli/docker_api_containers_test.go (hunk 5 above)
Comment (AntaresS) [reply integration-cli/docker_api_containers_test.go]: @thaJeztah @tiborvass OK here is what I did. Let's check for only API ≥ 1.32 for now. I added TODO for future reference.
```
if versions.GreaterThanOrEqualTo(testEnv.DaemonAPIVersion(), "1.32") {
      c.Assert(res.StatusCode, checker.Equals, http.StatusBadRequest)
}
```

File: integration-cli/docker_api_network_test.go [hunk 27]
```diff
@@ -64,7 +65,16 @@ func (s *DockerSuite) TestAPINetworkCreateCheckDuplicate(c *check.C) {
 	c.Assert(isNetworkAvailable(c, name), checker.Equals, true)
 
 	// Creating another network with same name and CheckDuplicate must fail
-	createNetwork(c, configOnCheck, http.StatusConflict)
+	isOlderAPI := versions.LessThan(testEnv.DaemonAPIVersion(), "1.34")
+	expectedStatus := http.StatusConflict
+	if isOlderAPI {
+		// in the early test code it uses bool value to represent
+		// whether createNetwork is expected to fail fo nor.
```
Comment (tiborvass) [inline integration-cli/docker_api_network_test.go]: @AntaresS little typo. We should also explain that this will check that the status is NOT equal to http.StatusCreated.

File: integration-cli/docker_api_network_test.go [hunk 28]
```diff
@@ -253,7 +267,11 @@ func createDeletePredefinedNetwork(c *check.C, name string) {
 			CheckDuplicate: true,
 		},
 	}
-	createNetwork(c, config, http.StatusForbidden)
+	expectedStatus := http.StatusForbidden
+	if versions.LessThan(testEnv.DaemonAPIVersion(), "1.34") {
+		expectedStatus = -http.StatusCreated
```
Comment (tiborvass) [inline integration-cli/docker_api_network_test.go]: @AntaresS comment missing here.

File: integration-cli/docker_api_network_test.go (hunk 27 above)
Comment (AntaresS) [reply integration-cli/docker_api_network_test.go]: Done.

File: integration-cli/docker_api_network_test.go (hunk 28 above)
Comment (AntaresS) [reply integration-cli/docker_api_network_test.go]: Done/

**Review by tiborvass — APPROVED** [review]

**Review by thaJeztah — APPROVED** [review]
LGTM
